DRAsyncOperations.podspec

```diff
@@ -0,0 +1,13 @@
+Pod::Spec.new do |spec|
+  spec.name             = 'DRAsyncOperations'
+  spec.version          = '0.1'
+  spec.license          = { :type => 'MIT' }
+  spec.homepage         = 'https://github.com/dmcrodrigues/DRAsyncOperations'
+  spec.authors          = { 'David Rodrigues' => 'https://twitter.com/dmcrodrigues' }
+  spec.summary          = 'Implementation of a concurrent NSOperation to abstract and help the creation of asynchronous operations.
+.'
+  spec.source           = { :git => 'https://github.com/dmcrodrigues/DRAsyncOperations.git', :tag => '0.1' }
+  spec.platform         = :ios, '7.0'
```

Comment:
I don't see a benefit in specify the platform because the absence of this field will provide the Pod to every platform.